### ISSUE TITLE
Restoring and exploring images created by "Apart" partclone GUI

### ISSUE THREAD

**Issue by shasheene** [rescuezilla/rescuezilla#230]
There is a simple open-source Partclone GUI (graphical user interface) named "[Apart](https://github.com/alexheretic/apart-gtk)":

!["Apart" GUI](https://raw.githubusercontent.com/alexheretic/apart-gtk/readme-images/apart-gtk-usage.gif?raw=true)

As shown in the demonstration video above, the "Apart" program operates on individual partitions (like the partclone utility itself): the user queues up each partition on their hard drive individually, and Apart doesn't appear backup the partition table. The program produces image files with names such as:

```
boot-2017-05-06T1641.apt.ext2.gz
swap-2017-05-06T1641.apt.dd.gz
main-usb-2017-05-06T1641.apt.f2fs.gz
```

It's fortunate the Apart GUI places an "apt" in the filename. This makes the file naming pattern is unique, without any risk of conflicting with any of the other imaging frontends supported by Rescuezilla.

Adding support for such images in Rescuezilla is trivial. **Ideally, all Apart's partclone images that are in one folder will appear as one "image" in the Rescuezilla user-interface** as this will make it easier for users.

Side note: Like FSArchiver format, the lack of Apart creating a partition table backup means that the user (whether they use Apart or Rescuezilla) will have to manually "map" their images to the correct destination partition.

**Comment by candrapersada** [comment]
what is the difference between "Apart" and "rescuezilla" regarding backup and restore individual partitions #145?

**Comment by shasheene** [comment]
Rescuezilla makes a backup of metadata, including the partition table. Apart is most suited for individual partition backup and restore onto the same disk, but it's not capable of full system backup and restore to different disks.

In your case you have like 20 different Linux partitions so Apart might even be more suitable for you compared to the user-interface in Rescuezilla v2.2, because as you have just noted I haven't yet implemented task #145 to add a 'deselect all' checkbox.

**Feel free to try Apart out and see how you like i**t. Rescuezilla v2.3 will support backup images created with Apart so once you have a bunch of Apart images you can try out Rescuezilla v2.3 and you'll be in a great position to give me feedback about how Rescuezilla v2.3 supports Apart images.

I'll try to get the 'deselect all' checkbox implemented for Rescuezilla v2.3 though. **Because Rescuezilla can and should be better than every single other open-source imaging tool ever made.**
